content/developer/howtos/company.rst

```diff
@@ -193,13 +193,13 @@ from the view if the user does not have access to multiple companies. This is te
 
 .. _howto/company/security:
 
-Security rules
+Security Rules
```

Comment:
```suggestion
Security rules
```
[Headings should actually use the sentence case.](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#titles-and-headings)